scripts/scil_fodf_bundleparc.py

```diff
@@ -63,23 +54,27 @@ def _build_arg_parser():
         formatter_class=RawTextHelpFormatter)
 
     parser.add_argument('in_fodf', type=str,
-                        help='fODF input of order 6 or above. If the input '
-                             'has more than 28 coefficients, only the first 28'
-                             ' will be used.')
-    parser.add_argument('in_wm', type=str,
-                        help='Whole-brain WM mask input.')
+                        help='fODF input.')
     parser.add_argument('--out_prefix', type=str, default='',
                         help='Output file prefix. Default is nothing. ')
-    parser.add_argument('--out_folder', type=str, default='',
-                        help='Output destination. Default is the current '
-                             'directory.')
+    parser.add_argument('--out_folder', type=str, default='bundleparc',
+                        help='Output destination. Default is [%(default)s].')
     parser.add_argument('--nb_pts', type=int, default=50,
                         help='Number of divisions per bundle. '
                              'Default is [%(default)s].')
+    parser.add_argument('--min_blob_size', type=int, default=50,
```

Comment:
Maybe mention this argument make sense for a adult brain in 1x1x1? (here or in docstring)